Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -223,12 +241,16 @@ - (void)ensureContentConstraints
     const CGFloat kMinTextViewHeight = ceil(self.inputTextView.font.lineHeight
         + self.inputTextView.textContainerInset.top + self.inputTextView.textContainerInset.bottom
         + self.inputTextView.contentInset.top + self.inputTextView.contentInset.bottom);
-    const CGFloat kMaxTextViewHeight = 100.f;
+    // Exactly 4 lines of text with default sizing.
+    const CGFloat kMaxTextViewHeight = 98.f;
```

Comment:
So this value doesn't reflect dynamic text - it seems strange to hardcode this value rather than derive it from something like `"UIFont with default text size".lineHeight * 4.f`.

Reply:
This is as I intended it. I don't think we want "exactly 4 lines", in the case of larger fonts - it would cover too much of the interface. 

I think we want "about 100 points worth", and made the slight optimization for the majority of our users who use default font size.